### DISCUSSION TITLE
Dual monitor configuration help

### DISCUSSION THREAD

**Discussion by hui3215** [prasanthrangan/hyprdots#737] [Q&A]
Hello, I now have two screens, one comes with the laptop and the other is an independent monitor. Now I want to turn the independent monitor into the main monitor and the laptop monitor into the secondary screen. However, I cannot switch wallpaper themes. I can't use obs to record the screen. Please solve it. This is my configuration.

```
monitor = HDMI-A-1,1920x1080@59,-1920x0,1
monitor = eDP-1,1920x1080@59,0x0,1

workspace=monitor:HDMI-A-1,1
workspace=monitor:eDP-1,2
```

**Comment by kRHYME7** [comment]
Can you elaborate more about the problem? also hw to reproduce. 

I recommend using nwg-displays as it has a GUI to manage monitors and gives options for assigning workspaces

**Reply by hui3215** [reply]
NVIDIA 1650
I'll try using nwg-displays first to see the effect.

**Reply by hui3215** [reply]
I installed nwg-displays, but it doesn't work, wlr-randr is missing, but this package doesn't install

**Reply by kRHYME7** [reply]
>Can you elaborate more about the problem? also hw to reproduce.



I see try to install wlr-randr  first.

**Reply by hui3215** [reply]
❯ yay wlr-randr
4 aur/gnome-randr-rust 0.1.1.44.g688e0c2-1 (+1 0.02) 
    xrandr for Gnome/wayland, on distros that don't support wlr-randr
3 aur/wlr-layout-ui-git r79.7aecc75-1 (+0 0.00) 
    Provides a simple graphical front end for wlr-randr
2 aur/wlr-randr-git 0.3.0+1+gb9fad41-1 (+4 0.00) 
    Utility to manage outputs of a Wayland compositor (latest git version)
1 aur/wlr-randr 0.3.1-1 (+35 3.18) 
    Utility to manage outputs of a Wayland compositor
==> 要安装的包 (示例: 1 2 3, 1-3 或 ^4)
==> 1
AUR Explicit (1): wlr-randr-0.3.1-1
:: PKGBUILD 是最新的，跳过下载: wlr-randr
  1 wlr-randr                        (构建文件已存在)
==> 清理哪些软件包的构建文件？
==> [N]没有 [A]全部 [Ab]中止 [I]已安装 [No]未安装 或 (1 2 3, 1-3, ^4)
==> 
  1 wlr-randr                        (构建文件已存在)
==> 显示哪些包的差异？
==> [N]没有 [A]全部 [Ab]中止 [I]已安装 [No]未安装 或 (1 2 3, 1-3, ^4)
==> 
==> 正在创建软件包：wlr-randr 0.3.1-1 (2024年01月15日 星期一 08时32分45秒)
==> 获取源代码...
  -> 正在下载 wlr-randr-0.3.1.tar.gz...
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0  1383    0     0    0     0      0      0 --:--:--  0:00:01 --:--:--     0
curl: (22) The requested URL returned error: 404
==> 错误： 无法下载 https://git.sr.ht/~emersion/wlr-randr/refs/download/v0.3.1/wlr-randr-0.3.1.tar.gz
    正在放弃...
 -> 下载源文件时出错: /home/hui/.cache/yay/wlr-randr 
	context: exit status 1 
	

:: (1/1) 正在解析 SRCINFO: wlr-randr
pub   rsa4096 2016-04-01 [SC] [有效至：2024-03-31]
      34FF9526CFEF0E97A340E2E40FDE7BE0E88F5E48
uid             [ 未知 ] Simon Ser <contact@emersion.fr>
uid             [ 未知 ] emersion <contact@emersion.fr>
uid             [ 未知 ] [unknown attribute of size 74]
uid             [ 未知 ] [unknown attribute of size 82]
sub   rsa4096 2016-04-01 [E] [有效至：2024-03-31]

==> 正在创建软件包：wlr-randr 0.3.1-1 (2024年01月15日 星期一 08时32分48秒)
==> 正在检查运行时依赖关系...
==> 正在检查编译时依赖关系
==> 获取源代码...
  -> 正在下载 wlr-randr-0.3.1.tar.gz...
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0  1383    0     0    0     0      0      0 --:--:--  0:00:01 --:--:--     0
curl: (22) The requested URL returned error: 404
==> 错误： 无法下载 https://git.sr.ht/~emersion/wlr-randr/refs/download/v0.3.1/wlr-randr-0.3.1.tar.gz
    正在放弃...
 -> 生成时出错: wlr-randr-exit status 1
 -> 无法安装以下软件包, 需要手动介入处理:
wlr-randr - exit status 1

**Reply by kRHYME7** [reply]
I see, Try this command first ``` yay -Sy nwg-displays wlr-randr     ``` 


nwg-displays looks like this 
![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/5abecb40-bb66-4e60-b26d-d1211d7bc9f1)
